Makefile

```
@@ -202,9 +205,28 @@ local-cluster: # Create a local kind cluster
 release: ## Prepares a tagged release of the operator.
 .PHONY: release
 release:
-	./hack/release/make-release-tag.sh $(OLD_VERSION) $(NEW_VERSION)
+	./hack/release/make-release-tag.sh "$(OLD_VERSION) $(NEW_VERSION)"
 
 test-e2e: ## Runs e2e tests.
 .PHONY: test-e2e
 test-e2e: deploy
 	go test -timeout 20m -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
+
+# OLM
+.PHONY: bundle
+bundle: manifests
+	operator-sdk generate kustomize manifests -q
+	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version "$(SEMVER)"
+	operator-sdk bundle validate ./bundle
+
+.PHONY: bundle-build
+bundle-build:
+	"$(CONTAINER_ENGINE)" build -t "$(IMAGE)"-bundle:"$(VERSION)" -f bundle.Dockerfile
+
+.PHONY: bundle-push
+bundle-push:
+	"$(CONTAINER_ENGINE)" push "$(IMAGE)"-bundle:"$(VERSION)"
+  
+.PHONY: bundle-push-dev
+bundle-push-dev:
+	"$(CONTAINER_ENGINE)" push "$(AUTH) $(IMAGE)"-bundle:"$(VERSION)"
```

I think you want word-splitting for `$(AUTH)` so that an empty value is elided instead of being treated as a parameter:
```suggestion
	"$(CONTAINER_ENGINE)" push $(AUTH) "$(IMAGE)-bundle:$(VERSION)"
```